package/Dockerfile

```diff
@@ -15,10 +15,14 @@ RUN mkdir -p /go/src/github.com/k3s-io/helm-set-status && \
     make -C /go/src/github.com/k3s-io/helm-set-status install
 
 FROM alpine:3.12
-RUN apk add -U --no-cache ca-certificates jq bash git
+RUN apk add -U --no-cache ca-certificates jq bash git && \
+    adduser -D -u 1000 -s /bin/bash klipper-helm
+WORKDIR /home/klipper-helm
 COPY --from=extract /usr/bin/helm_v2 /usr/bin/helm_v3 /usr/bin/tiller /usr/bin/
 COPY --from=plugins /root/.local/share/helm/plugins/ /root/.local/share/helm/plugins/
 COPY entry /usr/bin/
 ENTRYPOINT ["entry"]
+ENV HOME=/home/klipper-helm
```

Comment:
I don't believe this is necessary; it should be set by the shell since you have a proper user account created by adduser.
```suggestion
```

Reply:
Oh right! That came from helm requiring the `$HOME` env. `adduser` also adds this so no longer needed. Thank you.